### ISSUE TITLE
Ender's pearls are erased in creative

### ISSUE THREAD

**Issue by MrBlasyMSK** [ItsMax123/AntiGlitch#8]
If you throw ender pearls in creative they are erased, I did it by throwing them to the ceiling and they were erased

The version is the last of everything

**Comment by ItsMax123** [comment]
I tried it and it works for me... Do you have any other plugin that might be doing that?

**Comment by MrBlasyMSK** [comment]
No, and when deleting your plugin it worked fine, it's weird

**Comment by ItsMax123** [comment]
only in creative? could you possibly take a video of it?

**Comment by MrBlasyMSK** [comment]
Yes, I'll send you a video later and I will put the updated version

**Comment by MrBlasyMSK** [comment]
https://streamable.com/8d9cvj

Latest version dev 13
